src/controllers/trainee/routes.ts

```diff
@@ -3,10 +3,9 @@ import { authMiddleWare, validationHandler } from '../../libs';
 import { traineeRoutes } from './Controller';
 import { default as validation } from './validation';
 const traineeRouter = express.Router();
-traineeRouter.route('/:id?')
+traineeRouter.route('/')
                 .get(validationHandler(validation.get), authMiddleWare('getUsers', 'read'), traineeRoutes.get)
                 .post(validationHandler(validation.create), traineeRoutes.create)
```

Comment:
Why are we not using auth middleware here?

Reply:
I have added auth middleware to all API requests